modules/phase_field/include/kernels/SoretDiffusion.h

```diff
@@ -52,6 +53,9 @@ class SoretDiffusion : public Kernel
 
   /// Boltzmann constant
   const Real _kb;
+
+  /// Shape function
+  Function * const _f_shape;
```

Comment:
If you revise this anyways could you please keep the const member ```_kb``` at the bottom of the class (and the initializer list). This is a lose (unwritten) convention that we have th the phase field classes.